packages/@o3r/eslint-config/schematics/ng-add/index.ts

```diff
@@ -0,0 +1,113 @@
+import type { JsonObject } from '@angular-devkit/core';
+import { chain, Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
+import type { NgAddSchematicsSchema } from './schema';
+import * as path from 'node:path';
+import { updateLinterConfigs } from './linter';
+import { readFileSync } from 'node:fs';
+
+const reportMissingSchematicsDep = (logger: { error: (message: string) => any }) => (reason: any) => {
+  logger.error(`[ERROR]: Adding @o3r/eslint-config has failed.
+You need to install '@o3r/schematics' package to be able to use the eslint-config package. Please run 'ng add @o3r/schematics'.`);
+  throw reason;
+};
+
+/**
+ * Add Otter eslint-config to an Angular Project
+ * @param options
+ */
+function ngAddFn(options: NgAddSchematicsSchema): Rule {
+  /* ng add rules */
+  return async (tree: Tree, context: SchematicContext) => {
+    const eslintPackageJsonPath = require.resolve(`eslint/package.json`);
+    const eslintInstalledVersion = JSON.parse(readFileSync(eslintPackageJsonPath, { encoding: 'utf8' })).version;
+    const { lt } = await import('semver');
+    if (lt(eslintInstalledVersion, '9')) {
```

Comment:
using this package might be ok with eslint@^8 if flat config is enabled
also in the warning I would recommend to upgrade first to flat config and only mention `@o3r/eslint-config-otter` as a temporary solution